src/main/resources/locale.properties

```diff
@@ -305,3 +300,6 @@ words.refMessage=
 words.reputation=
 words.role=
 words.unknown=
+command.repSettings.embed.choose=
```

Comment:
Could we get this sorted correctly?
(Applies to all locale files)

Reply:
Sorted